oscars.iml

```diff
@@ -71,6 +73,5 @@
     <orderEntry type="library" scope="TEST" name="Maven: com.vaadin.external.google:android-json:0.0.20131108.vaadin1" level="project" />
     <orderEntry type="library" scope="TEST" name="Maven: org.springframework:spring-test:5.2.10.RELEASE" level="project" />
     <orderEntry type="library" scope="TEST" name="Maven: org.xmlunit:xmlunit-core:2.7.0" level="project" />
-    <orderEntry type="library" name="gson-2.8.5" level="project" />
```

Comment:
Also it seems that your .iml file removed the gson library?

Reply:
Nevermind, I see it at the top of the file now. It says there are conflicts on this file so I'm wondering if that is it?